### PR TITLE
Add arabic language support

### PR DESCRIPTION
**Pull request by linuxscout**
add arabic code
add arabic module language
create a gammar rules

### CODE REVIEW

File: languagetool-language-modules/ar/src/main/resources/org/languagetool/resource/ar/tagset.txt [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                [person]    1       = first person
+                            2       = second person
+                            3       = third person
+                
+                [number]    s       = singular
+                            p       = plural
+                            
+    Exceptions:
+        avoir               “V avoir”  instead of  “V”
+        être                “V etre”   instead of  “V”
```
Comment (dpelle) [inline languagetool-language-modules/ar/src/main/resources/org/languagetool/resource/ar/tagset.txt]: I don't speak Arabic, but these are obviously French verbs.
It looks like a copy/paste from the French tagset.txt without changing it for Arabic.

File: languagetool-language-modules/ar/src/main/java/org/languagetool/language/Arabic.java [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+  private Tagger tagger;
+  //private Chunker chunker;
+  private SentenceTokenizer sentenceTokenizer;
+  private Synthesizer synthesizer;
+  private Disambiguator disambiguator;
+  private WordTokenizer wordTokenizer;
+  private LuceneLanguageModel languageModel;
+
+  /**
+   * @deprecated use {@link AmericanArabic} or {@link BritishArabic} etc. instead -
```
Comment (dpelle) [inline languagetool-language-modules/ar/src/main/java/org/languagetool/language/Arabic.java]: AmericanArabic and BritishArabic? There're no such variant most certainly.

It looks like a global substitute mistake s/English/Arabic/ which changed AmericanEnglish and BritishEnglish into AmericanArabic and BritishArabic.

File: languagetool-language-modules/ar/src/main/resources/org/languagetool/resource/ar/tagset.txt (hunk 1 above)
Comment (linuxscout) [inline languagetool-language-modules/ar/src/main/resources/org/languagetool/resource/ar/tagset.txt]: yes, I doesn't implement the tagger yet, 
I think that this tagset will be well for arabic,

File: languagetool-language-modules/ar/src/main/resources/org/languagetool/resource/ar/arabic_tags.txt [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+V avoir sub impa 2 p
+V avoir sub impa 2 s
+V avoir sub impa 3 p
+V avoir sub impa 3 s
+V avoir sub pres 1 p
+V avoir sub pres 1 s
+V avoir sub pres 2 p
+V avoir sub pres 2 s
+V avoir sub pres 3 p
+V avoir sub pres 3 s
```
Comment (dpelle) [inline languagetool-language-modules/ar/src/main/resources/org/languagetool/resource/ar/arabic_tags.txt]: This looks like the French POS tags, not Arabic.

File: languagetool-language-modules/ar/src/test/java/org/languagetool/AppTest.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
+package org.languagetool;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest
```
Comment (danielnaber) [inline languagetool-language-modules/ar/src/test/java/org/languagetool/AppTest.java]: This looks like auto-generated code that can be removed?

**Comment by danielnaber** [comment]
Thanks for the pull requests. Could you clean up the pull request so it doesn't contain duplicated code from English? Also, what are your long-term plans in maintaining this? We can only accept new languages when we're confident that they'll be maintained in the long run.

**Comment by linuxscout** [comment]
I will fix this and retry

**Pull request closed without merging**